develop/application-devs/interact/understand-parachain-data.md

```diff
@@ -0,0 +1,388 @@
+---
+title: Understanding Metadata
+description: Learn how to interact and decipher parachain data (metadata).
+---
+
+Each Polkadot SDK-based network exposes information about its runtime, allowing developers to know granular details regarding the types, pallets, and runtime APIs that a chain may offer.  Metadata files are SCALE-encoded, and allow the development of browser-based, mobile, or desktop applications that can keep up with potential forkless upgrades.  The concept of metadata also aids in ensuring that a single application can support many Polkadot SDK-based chains at once.
+
+## Exposing runtime information as metadata
+
+To interact with a node or the information stored in the blockchain, you need to know how to connect to the chain and how to access the features the runtime exposes to the outside world.
+In general, this interaction involves a remote procedure call (RPC) to request information you're interested in retrieving or updating.
+An application developer typically needs to know about the contents of the runtime's logic, including the following details:
+
+- The version of the runtime that the application is connecting to.
+- The APIs that the runtime supports.
+- The pallets that are implemented for that specific runtime.
+- The functions and their type signatures that are defined for that specific runtime.
+- The custom types that are defined for that specific runtime.
+- The parameters that the runtime exposes for users to set.
+
+Because the Polkadot SDK is modular and provides a composable framework for building a blockchain, there's no predefined schema of properties.
+Instead, every runtime is configured with its own set of properties and those properties—including functions and types—can change over time with runtime upgrades.
+
+To capture the information that's unique to a runtime, the Polkadot SDK enables you to generate the runtime **metadata** schema.
+The metadata for a runtime describes the pallets in-use and types that are defined for a specific version of the runtime.
+
+For every pallet, the metadata includes information about its storage items, functions, events, errors, and constants.
+The metadata also includes type definitions for any custom types included in the runtime.
+
+Because it provides a complete inventory of the runtime, the metadata is the key to enabling client applications to interact with the node, parse responses, and format message payloads correctly per chain.
```

Comment:
```suggestion
Because it provides a complete inventory of the runtime of a chain, the metadata is the key to enabling client applications to interact with the node, parse responses, and format message payload with that chain.
```